ttnn/ttnn/operations/unary.py

```diff
@@ -430,7 +430,7 @@ def activation_function(
         return output_tensor
 
     if isinstance(activation_function, ttnn.decorators.Operation):
-        activation_function.__name__ = f"ttnn.{(name)}"
+        activation_function.decorated_function.__name__ = f"ttnn.{(name)}"
```

Comment:
I like the change for adding `__name__` to the `ttnn` operations but I don't think this MR can be merged as is. Why can't this change be done at `register_operation` instead of updating op files with every line? Also, this update is not currently reflected in all ops and I would rather have this done whole-sale instead of having this annotation on partial set of ops.

Reply:
That's a fair point. I noticed the `__name__` and `decorated_function.__doc__` were all individually assigned like that, so I didn't want to make any drastic changes. However, these individual changes are only applicable when `fast_runtime_mode` is false. When `fast_runtime_mode` is true, the changes are reflected whole-sale under `register_operation`. Since `fast_runtime_mode` is true by default, I can split this commit into two and address this issue at a later time. Would that be ok?

Reply:
Can you create an issue to follow-up on this? That sounds good to me.

Reply:
@arakhmati made a separate PR here: [#98](https://github.com/tenstorrent/tt-metal/pull/9730)

Reply:
great, thanks!